packages/core/root/core-root-server-internal/src/server.ts

```diff
@@ -9,6 +9,8 @@
 import apm from 'elastic-apm-node';
 import { reportPerformanceMetricEvent } from '@kbn/ebt-tools';
 import type { Logger, LoggerFactory } from '@kbn/logging';
+import type { NodeRoles } from '@kbn/core-node-server';
+import { CriticalError as CriticalException } from '@kbn/core-base-server-internal';
```

Comment:
I was looking for the `CriticalException` implementation and couldn't find it. Almost missed the import rename. 

Nice troll, but we should probably remove the rename :trollface: .

Reply:
I honestly just thought it read a bit nicer inline. Happy to remove if it will save others a couple of miniutes.